forte/data/types.py

```diff
@@ -20,8 +20,13 @@
 __all__ = [
     "ReplaceOperationsType",
     "DataRequest",
+    "DATA_INPUT",
+    "DATA_OUTPUT"
 ]
 
 ReplaceOperationsType = List[Tuple[Span, str]]
 
 DataRequest = Dict[Type[Entry], Union[Dict, List]]
+
+DATA_INPUT = 0
```

Comment:
This file is used to define a couple "complex type" for type annotation purpose. I think `DATA_INPUT` and `DATA_OUTPUT` are not in the right place?

Reply:
Now `DATA_INPUT/DATA_OUTPUT` are static members of class `TrainPreprocessor`.